src/packages/migrate/src/__tests__/handlePanic.test.ts

```diff
@@ -79,4 +110,92 @@ describe('handlePanic', () => {
       )
     }
   })
+
+  // Only works locally (not in CI)
+  it.skip('test interactive engine panic', async () => {
```

Comment:
Probably you want to remove the `.skip`?

Reply:
This wouldn't have worked in the CI. I have change the test to expect a different result if it is run in the CI